seed/challenges/01-front-end-development-certification/html5-and-css.json

```diff
@@ -1691,91 +1695,6 @@
       "challengeType": 0
     },
     {
-      "id": "bad87fee1348bd9aedf08818",
```

Comment:
I don't quite understand the diff, I see some block of code removed and nothing added. Bit weird.